Controller/Application/Offers.php

```diff
@@ -35,45 +26,18 @@ class Offers extends Action
      */
     private $jsonFactory;
 
-    /**
-     * @var Session
-     */
-    private $checkoutSession;
-
-    /**
-     * @var ComperiaApplicationFactory
-     */
-
-    private $comperiaApplicationFactory;
-    /**
-     * @var LoggerInterface
-     */
-    private $logger;
-
     /**
      * Index constructor.
```

Comment:
wyśmienicie
ja bym i to wywalił, kolejne 2 linie mniej
https://pbs.twimg.com/media/Eey7iwYX0AAwO_E.jpg

Reply:
a właściwie, to nawet błędny ten komentarz, bo to konstruktor `Offers`, a nie `Index`